test/pieces-pages.js

```diff
@@ -162,4 +171,36 @@ describe('Pieces Pages', function() {
     });
   });
 
+  it('pieces-page as home page: switch page types', function() {
+    return Promise.try(function() {
+      return apos.docs.db.update({
+        slug: '/'
+      }, {
+        $set: {
+          type: 'events'
+        }
+      });
+    }).then(function() {
+      return apos.docs.db.update({
+        slug: '/events'
+      }, {
+        $set: {
+          type: 'default'
+        }
+      });
+    });
+  });
+
+  it('pieces-page as home page: correct permalinks on index page', function(done) {
+    return request('http://localhost:7900/', function(err, response, body) {
+      assert(!err);
+      // Is our status code good?
+      assert.equal(response.statusCode, 200);
+      // Only page one events should show up
+      assert(body.match(/event-001"/));
```

Comment:
perhaps a silly question, but what is the unpaired double quote in this line and also in the line below?

Reply:
This one is OK, it is just that I don't care about matching the entire URL here.